modules/mesh-task/variables.tf

```diff
@@ -34,7 +36,7 @@ variable "consul_image" {
 variable "consul_ecs_image" {
   description = "consul-ecs Docker image."
   type        = string
-  default     = "docker.mirror.hashicorp.services/hashicorp/consul-ecs:0.1.2"
+  default     = "ishustava/consul-ecs-dev:latest"
```

Comment:
heads up that I've found docker hub images heavily rate limited and ended up using ghcr.